core/client/fs/src/main/java/alluxio/client/block/policy/LocalFirstAvoidEvictionPolicy.java

```diff
@@ -78,9 +83,12 @@ public WorkerNetAddress getWorker(GetWorkerOptions options) {
   }
 
   /**
-   * The information of BlockWorkerInfo is update after a file complete write. To avoid evict,
-   * user should configure "alluxio.user.block.avoid.eviction.policy.reserved.size.bytes"
-   * to reserve some space to store the block.
+   * Calculate the available bytes for a worker with the added buffer of
+   * USER_FILE_WRITE_AVOID_EVICTION_POLICY_RESERVED_BYTES
+   * (alluxio.user.block.avoid.eviction.policy.reserved.size.bytes)
+   *
+   * Since the information of BlockWorkerInfo is updated <em>after</em> a file
+   * completes a write, mCapacityBytes minus mUsedBytes may not be the true available bytes.
```

Comment:
Fact-check me on this please; I just updated this according to the prior description.